lib/Util/ArrayColumn.php

```diff
@@ -8,11 +8,12 @@
  * @author Ben Ramsey (http://benramsey.com)
  */
 function array_column($input = null, $column_key = null, $index_key = null) {
+  $argc = func_num_args();
```

Comment:
Can you please explain why did you have to make this change?

Reply:
I've updated also patch versions of other deps and phpcs.
The phpcs was rising an error here see: https://circleci.com/gh/mailpoet/mailpoet/17601